Orange/widgets/utils/tests/test_itemmodels.py

```diff
@@ -310,3 +310,12 @@ def test_filtering(self):
                             skip_hidden_vars=False)
         model.set_domain(Domain(attrs))
         self.assertEqual(list(model), disc)
+
+    def test_no_separators(self):
+        attrs = [ContinuousVariable(n) for n in "abg"]
+        classes = [ContinuousVariable(n) for n in "deh"]
+        metas = [ContinuousVariable(n) for n in "ijf"]
+
+        model = DomainModel(order=DomainModel.SEPARATED, separators=False)
+        model.set_domain(Domain(attrs, classes, metas))
+        self.assertEqual(list(model), classes + metas + attrs)
```

Comment:
What about checking whether it works with `separators=True`?

Reply:
I actually added this test before you responded.